tests/models/qwen2_vl/test_modeling_qwen2_vl.py

```diff
@@ -428,11 +429,16 @@ def test_small_model_integration_test_batch_different_resolutions(self):
                 ],
             }
         ]
-        text2, vision_infos2 = self.processor.apply_chat_template(
-            messages2, tokenize=False, add_generation_prompt=True
-        )
+        text2 = self.processor.apply_chat_template(messages2, tokenize=False, add_generation_prompt=True)
+
+        vision_infos = self.extract_vision_info(messages2)
+        image_url = vision_infos[0]["image"]
+        image_input2 = Image.open(requests.get(image_url, stream=True).raw)
```

Comment:
I think that the processor is supposed to be able to handle urls or images and properly open them, would make sense to add this if it's not currently the case for an eased usage no? 🤗